poker-pals/src/components/Tables.vue

```diff
@@ -23,7 +23,23 @@
                 <p> {{room.tableName}}</p>
             </div>
             <div class = "seats" id = "seatsId">
-            <!-- TODO: Update seat availability as seats fill up.--> 
+                <p> {{room.numberOfFullSeats}}/{{room.numberOfTableSeats}}</p>
+            </div>
+        </div>
+      </div>
+      <!-- Display full table --> 
+        <!-- Join room on click --> 
+      <div class="table-full" id = "table-click" v-else v-on:click="joinRoom(room)">
```

Comment:
What does this `v-else` do?

Reply:
v-else is used to change the appearance of the table once full.

Reply:
Nevermind, there's a `v-if` at the top